app/src/main/java/org/mifos/mobile/ui/home/HomeScreen.kt

```diff
@@ -0,0 +1,303 @@
+            .padding(20.dp),
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        userBitmap?.let {
+            MifosUserImage(
+                isDarkTheme = isSystemInDarkTheme(),
+                modifier = Modifier
+                    .size(84.dp)
+                    .clickable(
+                        indication = null,
+                        interactionSource = interactionSource,
+                    ) { userProfile.invoke() },
+                bitmap = userBitmap
+            )
+        }
+        Text(
+            text = stringResource(
+                R.string.hello_client,
+                username
+            ),
+            style = MaterialTheme.typography.headlineSmall,
+            color = if (isSystemInDarkTheme()) Color.White else Color.Black,
+            modifier = Modifier
+                .padding(horizontal = 20.dp)
+                .fillMaxWidth(1f)
+        )
+    }
+}
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+private fun HomeCard(
+    modifier: Modifier,
+    titleId: Int,
+    drawableResId: Int,
+    onClick: () -> Unit
+) {
+    Card(
+        modifier = modifier,
+        onClick = { onClick.invoke() }
+    ) {
+        Column(
+            modifier = Modifier
+                .padding(8.dp)
+                .fillMaxWidth(),
+            horizontalAlignment = Alignment.CenterHorizontally,
+            verticalArrangement = Arrangement.Center
+        ) {
+            Icon(
+                painter = painterResource(id = drawableResId),
+                contentDescription = null,
+                modifier = Modifier.size(56.dp),
+                tint = MaterialTheme.colorScheme.primary
+            )
+            Text(
+                text = stringResource(id = titleId),
+                textAlign = TextAlign.Center,
+                style = MaterialTheme.typography.bodyMedium,
+                color = if (isSystemInDarkTheme()) Color.White else Color.Black
+            )
+        }
+    }
+}
+
+@Composable
+private fun AccountOverviewCard(
+    totalLoanAmount: Double,
+    totalSavingsAmount: Double,
+    totalSavings: () -> Unit,
+    totalLoan: () -> Unit
+) {
+    val context = LocalContext.current
+
+    Row {
+        Card(
+            modifier = Modifier
+                .fillMaxWidth(),
+            colors = CardDefaults.cardColors()
+        ) {
+            Column(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(20.dp)
+            ) {
+                Text(
+                    text = stringResource(id = R.string.accounts_overview),
+                    style = MaterialTheme.typography.bodyMedium,
+                    color = if (isSystemInDarkTheme()) Color.White else Color.Black
+                )
+
+                Divider(
+                    modifier = Modifier.padding(top = 8.dp, bottom = 4.dp),
+                    color = if (isSystemInDarkTheme()) Color.White else Color.Black
```

Comment:
Move this in MifosTheme and choose current theme divider color that you create in theme